vercel.json

```diff
@@ -0,0 +1,3 @@
+{
+	"bunVersion": "1.3"
```

Comment:
```suggestion
	"bunVersion": "1.x"
```
fyi, its meant to just be `1.x` as vercel manages the minor & patch versions

see https://vercel.com/docs/functions/runtimes/bun#configuring-the-runtime